test/clientValidationTasks/crabConfiguration.py

```diff
@@ -17,7 +17,7 @@
 config.Data.splitting = 'FileBased'
 config.Data.unitsPerJob = 10
 config.Data.publication = True
-config.Data.outputDatasetTag = 'CRAB3_tutorial_May2015_MC_analysis'
+config.Data.outputDatasetTag = 'CRAB3_analysis'
```

Comment:
better not to use always the same dataset name. I'd chance once/month e.g. by putttin YYMM in the output DatasetTag

Reply:
thank you, i did the same as you have for status tracking tasks: 
https://github.com/ddaina/CRABServer/blob/6dcf3e474c9f5cf251b3371af66baa659a735a76/test/statusTrackingTasks/HC-splitByFile.py#L24

Reply:
there is a tension between "test publishing to a new dataset" and "test publishing to an existing dataset". In the tracking tasks I preferred the former, but it is good to do some of the latter as well, so I would stay with YYMM in this.

Reply:
thanks, done.